### DISCUSSION TITLE
如何关闭打包下载？

### DISCUSSION THREAD

**Discussion by caixiaohaier** [AlistGo/alist#1246] [Q&A]
我是在腾讯轻量云搭建的程序，带宽和容量都有限制，我觉得打包下载好像是程序从网盘下载原文件到服务器上面，然后在传给用户端的，也就是流量在服务器中转一次，所以有此想法，如果不对麻烦纠正流量走向是怎么走的？在官方文档和网上翻了一圈没看到有相关说明，以及设置里面也没有关闭打包下载的按钮。

**Comment by anwen-anyi** [comment]
你不用就行了呗QAQ~

**Comment by xhofe** [comment]
不是 不用担心，和单个文件下载走的流量相同。

**Reply by haoduck** [reply]
以后有没有可能添加开启或关闭的选项，貌似有些网盘用这个功能会出现CORS 错误(百度)，无法正常使用。

**Comment by ialtone** [comment]
同样希望出个关闭打包下载功能。

**Reply by huyangnl** [reply]
关闭下载，关闭预览这些功能都需要有开关。请考虑增加。
